### ISSUE TITLE
Unable to search for manga with no authors

### ISSUE THREAD

**Issue by muckas** [nixonjoshua98/manganelo#17]
Hi. Seems like if manga has no authors, search results in an error.
As an example here "Shingeki No Kyojin - Birth Of Levi"
https://manganato.com/search/story/attack_on_titan

```
Python 3.10.5 (main, Jun  6 2022, 18:49:26) [GCC 12.1.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import manganelo
>>> manganelo.get_search_results("attack on titan")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/mnt/nvme/documents/projects/manganelo-test/venv/lib/python3.10/site-packages/manganelo/storysearch.py", line 19, in get_search_results
    return [SearchResult(ele) for ele in soup.find_all(class_="search-story-item")]
  File "/mnt/nvme/documents/projects/manganelo-test/venv/lib/python3.10/site-packages/manganelo/storysearch.py", line 19, in <listcomp>
    return [SearchResult(ele) for ele in soup.find_all(class_="search-story-item")]
  File "/mnt/nvme/documents/projects/manganelo-test/venv/lib/python3.10/site-packages/manganelo/models/searchresult.py", line 18, in __init__
    self.authors: list[str] = self._parse_authors(soup)
  File "/mnt/nvme/documents/projects/manganelo-test/venv/lib/python3.10/site-packages/manganelo/models/searchresult.py", line 35, in _parse_authors
    txt = soup.find("span", class_="text-nowrap item-author").text
AttributeError: 'NoneType' object has no attribute 'text'
```

**Comment by nixonjoshua98** [comment]
Ah thats annoying. I'll look at this later unless you want to submit a fix yourself

**Comment by muckas** [comment]
Found another issue with genres parsing, fixed both in PR #18

**Comment by nixonjoshua98** [comment]
Published here - https://pypi.org/project/manganelo/1.21.0/ - Thanks for fixing it!
